packages/bot-web-ui/src/assets/sass/flyout.scss

```diff
@@ -64,11 +64,12 @@
     }
     &__button {
         &-new {
-            width: 100%;
-            font-size: 100%;
-            box-shadow: 0 2px 8px 0 $COLOR_LIGHT_BLACK_2;
-            margin: $default-margin 0;
-            background-color: var(--general-main-1);
+            width: 25%;
+            height: 4rem !important;
+            font-size: var(--text-size-s);
+            font-weight: bold;
+            border-top-left-radius: 0rem !important;
+            border-bottom-left-radius: 0rem !important;
```

Comment:
Same here

Reply:
I need those important because I want to overwrite those styles come from Input styles and .dc-btn